arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -900,12 +900,12 @@ public async Task GetVertexAsync_ShouldThrow_WhenGraphIsNotFound()
 
             var ex = await Assert.ThrowsAsync<ApiErrorException>(async () =>
             {
-                await _client.GetVertexAsync<GetVertexMockModel>(graphName, vertex, "12345");
+                await _client.GetVertexAsync<GetVertexMockModel>(graphName, vertex, "");
             });
 
             Assert.True(ex.ApiError.Error);
             Assert.Equal(HttpStatusCode.NotFound, ex.ApiError.Code);
-            Assert.Equal(1924, ex.ApiError.ErrorNum); // ERROR_GRAPH_NOT_FOUND
+            Assert.Equal(1924, ex.ApiError.ErrorNum); // GRAPH_DUPLICATE
```

Comment:
`1924` is indeed `ERROR_GRAPH_NOT_FOUND` according to the docs: https://www.arangodb.com/docs/stable/appendix-error-codes.html